pod/perlunicode.pod

```diff
@@ -1817,50 +1817,157 @@ through C<0x10FFFF>.)
 
 =head2 Security Implications of Unicode
 
-First, read
-L<Unicode Security Considerations|https://www.unicode.org/reports/tr36>.
+The security implications of Unicode are quite complicated, as you might
+expect from it trying to handle all the world's scripts.  An
+introduction is in
+L<Unicode Security Methods|https://www.unicode.org/reports/tr39>.
 
-Also, note the following:
+Here are a few examples of pitfalls
 
 =over 4
 
-=item *
+=item Confusables
+
+Many characters in Unicode look similar enough to other characters that
+they could be easily confused with each other.  This is true even within
+the same script, for example English, where the digit C<0> and a capital
+letter C<O> may look like each other.  But people who use that script
+know to look out for that.
+
+In Unicode, a digit in one script may be confusable with a digit having
+a different value in another.  A malicious website could use this to
+make it appear that the price of something is less than it actually is.
+(You can use L<perlre/Script Runs> to make sure such digits are not
+being inter-mixed.)
+
+This is a general problem with internet addresses.  The people who give out
+domain names need to be careful to not give out ones that spoof other ones,
+(examples in L<perlre/Script Runs>).
```

Comment:
the rest of a sentence after a comma being parenthetical seems strange to me .

Reply:
Agreed; I'll fix before pushing